source/com.google.android.play/core.extensions/extensions-aar/src/main/java/xamarin/google/android/play/core/assetpacks/AssetPackStateUpdateListenerWrapper.java

```diff
@@ -0,0 +1,37 @@
+package xamarin.google.android.play.core.assetpacks;
+
+import com.google.android.play.core.assetpacks.AssetPackState;
+import com.google.android.play.core.assetpacks.AssetPackStateUpdateListener;
+
+public class AssetPackStateUpdateListenerWrapper {
```

Comment:
@jonpryor @jpobst are we happy with the naming convention I'm using here? I figure this will eventually be a pattern we roll out in a few places so do we like `Wrapper`? or should we use `Helper` or `Extension` or something else?

Reply:
Wrapper is good/better IMO

Reply:
👍  Wrapper seems fine.